packages/botonic-plugin-contentful/src/contentful/manage/manage-contentful.ts

```diff
@@ -60,17 +60,27 @@ export class ManageContentful implements ManageCms {
     context: ManageContext,
     contentId: ContentId,
     fieldType: ContentFieldType,
-    value: any
+    value: any,
+    code?: string
```

Comment:
I don't think we need a special argument for the code. We should only add a new ContentFieldType

Reply:
to avoid making 1 call to change the code per field, this method could validate if the previous value of the entry already has the desired value

Reply:
Actually it's not making a new call to change the Code, it's using the actual call that change the contents to change the Code too.